sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -0,0 +1,853 @@
+            AsyncServiceBusSharedKeyCredentialPolicy(self._endpoint, self._credential, "Authorization") \
+            if isinstance(self._credential, ServiceBusSharedKeyCredential) \
+            else AsyncBearerTokenCredentialPolicy(self._credential, JWT_TOKEN_SCOPE)
+        if policies is None:  # [] is a valid policy list
+            policies = [
+                RequestIdPolicy(**kwargs),
+                self._config.headers_policy,
+                self._config.user_agent_policy,
+                self._config.proxy_policy,
+                ContentDecodePolicy(**kwargs),
+                ServiceBusXMLWorkaroundPolicy(),
+                self._config.redirect_policy,
+                self._config.retry_policy,
+                credential_policy,
+                self._config.logging_policy,
+                DistributedTracingPolicy(**kwargs),
+                HttpLoggingPolicy(**kwargs),
+            ]
+        if not transport:
+            transport = AioHttpTransport(**kwargs)
+        return AsyncPipeline(transport, policies)
+
+    @classmethod
+    def from_connection_string(cls, conn_str, **kwargs):
+        # type: (str, Any) -> ServiceBusManagementClient
+        """Create a client from connection string.
+
+        :param str conn_str: The connection string of the Service Bus Namespace.
+        :rtype: ~azure.servicebus.management.aio.ServiceBusManagementClient
+        """
+        endpoint, shared_access_key_name, shared_access_key, _ = parse_conn_str(conn_str)
+        if "//" in endpoint:
+            endpoint = endpoint[endpoint.index("//")+2:]
+        return cls(endpoint, ServiceBusSharedKeyCredential(shared_access_key_name, shared_access_key), **kwargs)
+
+    async def _get_entity_element(self, entity_name, enrich=False, **kwargs):
+        # type: (str, bool, Any) -> ElementTree
+
+        with _handle_response_error():
+            element = cast(
+                ElementTree,
+                await self._impl.entity.get(entity_name, enrich=enrich, api_version=constants.API_VERSION, **kwargs)
+            )
+        return element
+
+    async def _get_subscription_element(self, topic_name, subscription_name, enrich=False, **kwargs):
+        # type: (str, str, bool, Any) -> ElementTree
+
+        with _handle_response_error():
+            element = cast(
+                ElementTree,
+                await self._impl.subscription.get(
+                    topic_name, subscription_name, enrich=enrich, api_version=constants.API_VERSION, **kwargs)
+            )
+        return element
+
+    async def _get_rule_element(self, topic_name, subscription_name, rule_name, **kwargs):
+        # type: (str, str, str, Any) -> ElementTree
+
+        with _handle_response_error():
+            element = cast(
+                ElementTree,
+                await self._impl.rule.get(
+                    topic_name, subscription_name, rule_name, enrich=False, api_version=constants.API_VERSION, **kwargs)
```

Comment:
I see the method  `_get_subscription_element`/`_get_entity_element` have `enrich` in the args list, while `_get_rule_element` doesn't accept `enrich` and set to False.

Is it by design?

Reply:
rule doesn't have runtime information.

Reply:
gotcha, so it's only for the runtime info. we could probably document that.